1/contrib/s2i/run

```diff
@@ -9,6 +9,29 @@
 source /usr/local/bin/jenkins-common.sh
 source /usr/local/bin/kube-slave-common.sh
 
+#display the available i386 and x86_64 java's to confirm their presence and version
+alternatives --display java | grep family | grep i386
+alternatives --display java | grep family | grep x86_64
```

Comment:
Originally debug, but then I left in cause I thought it might be useful when debugging the odd ball customer issue that could be a JDK issue ... channeling my prior existence as an app server developer where knowing the precise JDK version was important :-)